torch/ao/quantization/observer.py

```diff
@@ -446,7 +446,11 @@ def __init__(
         factory_kwargs=None,
         eps=torch.finfo(torch.float32).eps,
     ) -> None:
-
+        if qscheme != torch.per_tensor_symmetric and qscheme != torch.per_tensor_affine:
```

Comment:
nit: please use the utility functions to do these checks: https://github.com/pytorch/pytorch/blob/master/torch/ao/quantization/utils.py#L115-L122

Reply:
Changed.